css-images-4/Overview.bs

```diff
@@ -2022,7 +2024,9 @@ Color Stop “Fixup”</h4>
 		to obtain only 90px,
 		which is then distributed in the 1:2 ratio
 		dictated by the <<flex>> values.
-		
+
+		<img src="images/example33-2.png" style="height: 1em; width:400px;" alt="example 2">
```

Comment:
Like above, maybe better an SVG than a PNG.

Simply name it "stripes2" to avoid numbering issues when other example images are added.

And if you want it add an alternative text, here's an example description:
"Horizontal line with a total width of 400 pixels showing the output of 'stripes(red .1fr, green .2fr, blue 100px)'. The left-most part is a red stripe of 30 pixels length, followed by a green stripe of 60 pixels, a blue one of 100 pixels and a transparent stripe of 210 pixels.".